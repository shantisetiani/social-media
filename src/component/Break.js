import React from "react";

function Break({ height }) {
  return <div style={{ height: height }}></div>;
}

export default Break;
