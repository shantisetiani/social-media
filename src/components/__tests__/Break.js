import { cleanup, render } from "@testing-library/react";
import renderer from "react-test-renderer";
import Break from "../Break";

describe("UI Test Break component", () => {
  afterEach(cleanup);

  it("matches break snapshot", () => {
    const tree = renderer.create(<Break height={12} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
