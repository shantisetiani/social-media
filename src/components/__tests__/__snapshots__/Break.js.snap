// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI Test Break component matches break snapshot 1`] = `
<div
  style={
    Object {
      "height": "12px",
    }
  }
/>
`;
